components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -80,9 +82,21 @@ export const PrebuildDetailPage: FC = () => {
             }
         });
         logEmitter.on("logs-error", (err: ApplicationError) => {
+            const phase = prebuild?.status?.phase?.name;
+            // An overcomplicated check for getting a "failed" prebuild state, for which we do not store logs.
+            if (
+                phase &&
+                ([PrebuildPhase_Phase.FAILED, PrebuildPhase_Phase.TIMEOUT].includes(phase) ||
+                    (phase === PrebuildPhase_Phase.AVAILABLE && prebuild.status?.message))
```

Comment:
`phase === PrebuildPhase_Phase.AVAILABLE && prebuild.status?.message`

Will this situation happen?

Reply:
This should happen for most failed prebuilds where they are in the available state but have an error message attached. Similar to https://github.com/gitpod-io/gitpod/blob/f324ac73f5d8f9ae062cb0412d4fd448f4de0cdd/components/gitpod-db/src/typeorm/workspace-db-impl.ts#L1090-L1094